docs/markdown/recursor/settings.md

```diff
@@ -583,7 +583,7 @@ for most installations.
 * Available since: 3.2
 
 Maximum number of seconds to cache an item in the DNS cache, no matter what the
-original TTL specified.
+original TTL specified. The minimal value is capped to 15 since PowerDNS Recursor 4.1.0.
```

Comment:
suggest rewording this, perhaps like I reworded this PR title, to avoid confusing use of min vs. max.